### PR TITLE
update faq with default language

### PR DESCRIPTION
**Pull request by JantsoP**
Update FAQ on how to change default language of the instance? Can be modified if needed

### CODE REVIEW

**Comment by JantsoP** [comment]
Sorry, i ment @Gargron for the help with Minio install :)

**Comment by JantsoP** [comment]
Added new translation for import and export :)

**Comment by JantsoP** [comment]
Add Finnish translation for emails

File: docs/Using-Mastodon/FAQ.md [hunk 1]
```diff
@@ -36,8 +36,15 @@ While Mastodon is compatible with GNU social in terms of server to server commun
 
 Because Mastodon has been created from a blank slate, it is much simpler to have the API mirror internal structures as closely as possible, rather than build an emulation layer. Secondly, the GNU social client API is actually a half-way implementation of the legacy Twitter API - that's the reason why it works with some older Twitter client apps. However, many of those apps are not maintained anymore, the GNU social API does not actually keep up with the real Twitter API and never fully implemented all its features; at the same time, the Twitter API was never meant for a federated service and so obscures some of the functionality.
 
+#### How i can change the default language of the Instance?
```
Comment (Gargron) [inline docs/Using-Mastodon/FAQ.md]: Either remove in favour of #1145 being self-described in the .env file, or refer to it. Modifying `i18n.default_locale` is a bad idea in this application, here `:en` is the most complete localization.

File: .env.production.sample [hunk 2]
```diff
@@ -44,6 +44,16 @@ SMTP_FROM_ADDRESS=notifications@example.com
 # S3_PROTOCOL=http
 # S3_HOSTNAME=192.168.1.123:9000
 
+# S3 (Minio Config (optional))
+# S3_ENABLED=true
+# S3_BUCKET=check minio's first launch for details
```
Comment (Gargron) [inline .env.production.sample]: I suggest moving "check minio's details etc" text into a comment line/block. The commented out variables should still contain valid (or empty) values.

**Review by Gargron — CHANGES_REQUESTED** [review]

**Comment by JantsoP** [comment]
All issues fixed?

**Review by Gargron — APPROVED** [review]

**Pull request merged**